### PR TITLE
Update Python installation instructions in docs

### PR DESCRIPTION
**Pull request by timmens**

### CODE REVIEW

File: docs_template/source/getting_started/preparing_your_system.md [hunk 1]
```diff
@@ -3,14 +3,10 @@
 Make sure you have the following programs installed and that these can be found on your
 path. This template requires
 
-- [Miniconda](https://docs.conda.io/en/latest/miniconda.html) or
-  [Micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html) or
-  Anaconda. Windows users: please consult {ref}`windows_user`
+- Python, which you must install using the installation instructions
+  [here](https://effective-programming-practices.vercel.app/python_installation_execution/index.html).
```
Comment (hmgaudecker) [inline docs_template/source/getting_started/preparing_your_system.md:7]: Please link directly to the pages for Windows, MacOS, Linux, and WSL for the adventurous.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -34,6 +34,7 @@ jobs:
           environment-file: environment.yml
           create-args: python=${{ matrix.python-version }}
           cache-environment: true
+          micromamba-version: 1.5.6-0
```
Comment (hmgaudecker) [inline .github/workflows/main.yml:37]: Looks like this needs to be restricted to Windows.

**Review by hmgaudecker — APPROVED** [review]
Great, thanks for the quick fix!!! 

I got rid of all traces of Anaconda and some more stuff -- please double-check once more.

**Comment by timmens** [comment]
> I got rid of all traces of Anaconda and some more stuff -- please double-check once more.

Checked!

Also, the CI error is gone after re-running the checks.

**Pull request merged**